### PR TITLE
Implement incidenttypes components

### PR DESCRIPTION
**Pull request by mkumbobeaty**

### CODE REVIEW

File: src/Incidents/components/IncidentTypes/ActionBar/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          className="actionButton"
+          size="large"
+        />
+      </Col>
+
+      <Col span={1} xl={1}>
+        <Button
+          type="circle"
+          icon="cloud-download"
+          title="Export selected incidenttypes"
```
Comment (BenMaruchu) [inline src/Incidents/components/IncidentTypes/ActionBar/index.js]: This is a title supposed to be `Export selected Incident Types`

Comment (BenMaruchu) [reply src/Incidents/components/IncidentTypes/ActionBar/index.js]: So for other places for title attribute change `incidenttypes`  to be a normal word

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
@mkumbobeaty hey there a some few quick update for this PR thanks

File: src/Incidents/components/IncidentTypes/ActionBar/index.js (hunk 1 above)
Comment (mkumbobeaty) [reply src/Incidents/components/IncidentTypes/ActionBar/index.js]: oky will fix that

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
@mkumbobeaty  sorry for disturbing but can you make first letters for incident types caps i.e `Incident Types` for titles

**Comment by BenMaruchu** [comment]
Also there is a conflict on this branch please pull changes first before pushing @mkumbobeaty

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**